packages/components/src/components/Icon/Icon.tsx

```diff
@@ -77,7 +62,8 @@ export function Icon(props: IconProps) {
       'q icon',
       `icon--size${iconSize}`,
       children.props.className,
-      styleProps.className,
+      props.className,
```

Comment:
I think we should probably also spread `otherProps.style` here

Reply:
Done too!